includes/class-wc-calypso-bridge-addons-screen.php

```diff
@@ -0,0 +1,79 @@
+<?php
+class WC_Calypso_Bridge_Addons_Screen extends WC_Admin_Addons {
```

Comment:
There are some linter/coding style issues with both of these new files.

From the root directory, you can run `composer install` (see https://github.com/Automattic/wc-calypso-bridge#getting-started) and it should create pre-commit hooks for you to catch these. You can also lint the file manually:

`$ ./vendor/bin/phpcs includes/class-wc-calypso-bridge-addons.php`
`$ ./vendor/bin/phpcs includes/class-wc-calypso-bridge-addons-screen.php`